central/clusterinit/backend/backend_test.go

```diff
@@ -390,16 +390,16 @@ func (s *clusterInitBackendTestSuite) TestCheckAccess() {
 		shouldFail  bool
 		expectedErr error
 	}{
-		"read access to both ServiceIdentity and APIToken should allow read access": {
+		"read access to both ServiceIdentity and Integration should allow read access": {
```

Comment:
same applies for other test cases
```suggestion
		"read access to both Administration and Integration should allow read access": {
```